upload/upload.go

```diff
@@ -54,41 +53,63 @@ func loadConfigFile(path string) (*Config, error) {
 
 	cfg, err := ini.Load(path)
 	if err != nil {
-		return config, nil
+		return config, err
 	}
 
 	if err := cfg.Section("").MapTo(config); err != nil {
 		return nil, err
 	}
 
+	if config.AccessKey == "" || config.SecretKey == "" || config.AccessToken == "" {
+		return nil, errors.New("failed to find credentials in configuration file")
+	}
+
+	if config.HostBase == "" {
+		config.HostBase = "inbox.test.ega.nbis.se"
+	}
+
+	if config.UseHttps {
+		config.HostBase = "https://" + config.HostBase
+	}
+
+	if config.Encoding == "" {
+		config.Encoding = "UTF-8"
+	}
+
+	if config.MultipartChunkSizeMb == 0 {
```

Comment:
```suggestion
	if config.MultipartChunkSizeMb <= 15 {
```

Reply:
Why less than 15 here? With 0 I make sure I give a value, if it's not set in the config file

Reply:
Library default is 15Mb and max filesize is 10000 x MultipartChunkZise

Reply:
If multipart size is not in the config file it will be set to 15Mb by the library, so we only needs to guard ourselves from the case when the config file has a value less than 15